lib/core/Head.js

```diff
@@ -16,13 +16,19 @@ class Head extends React.Component {
       if (link.blog) hasBlog = true;
     });
 
-    const highlightDefaultVersion = '9.12.0';
-    const highlightConfig = this.props.config.highlight || {
-      version: highlightDefaultVersion,
+    let highlight = {
+      version: '9.12.0',
       theme: 'default',
+      ...this.props.config.highlight,
     };
-    const highlightVersion = highlightConfig.version || highlightDefaultVersion;
-    const highlightTheme = highlightConfig.theme || 'default';
+
+    if (highlight.theme.indexOf('/') > -1) {
+      highlight = highlight.theme;
```

Comment:
Define a new variable `highlightURL` instead. Don't reassign a value of a different type to the `highlight` variable.